package/yast2-packager.changes

```diff
@@ -1,32 +1,36 @@
 -------------------------------------------------------------------
-Mon Jan 30 10:57:14 UTC 2017 - igonzalezsosa@suse.com
+Wed Feb  1 16:04:24 UTC 2017 - jreidinger@suse.com
 
+changes required for CaaSP:
 - Packages module is able to perform a package selection proposal
   during system upgrade (bsc#1009834)
-- 3.1.119.4
-
--------------------------------------------------------------------
-Tue Jan 24 13:15:46 UTC 2017 - jsrain@suse.cz
-
 - do not check content file on media if there is none in the
   ramdisk (fate#322276)
-- 3.1.119.3
-
--------------------------------------------------------------------
-Wed Dec 21 15:06:31 CET 2016 - schubi@suse.de
-
 - Removed: Copy /etc/zypp/zypp.conf from inst_sys to target system.
   Instead: Patching /etc/zypp/zypp.conf in the installed system
            directly.
   (FATE#321764)
-- 3.1.119.2
+- Copy /etc/zypp/zypp.conf from inst_sys to target system.
+  (FATE#321764)
+- 3.1.121.1
 
 -------------------------------------------------------------------
-Thu Dec  8 17:56:24 CET 2016 - schubi@suse.de
+Wed Feb  1 16:03:24 UTC 2017 - jreidinger@suse.com
 
-- Copy /etc/zypp/zypp.conf from inst_sys to target system.
-  (FATE#321764)
-- 3.1.119.1
+- Fix escaping spaces ( yast uses web form escaping to "+" but
+  libzypp expects percentage escaping to "%20" ) (bsc#954813)
+- Fix also valid showing valid libzypp formats of iso URL like
+  iso:/test?iso=test.iso&path=url%3A%2Finstall
+- Fix editing of iso:/ without triple slash that previously caused
+  lost of first element of path
+- 3.1.121
+
+-------------------------------------------------------------------
+Mon Jan 30 10:57:14 UTC 2017 - igonzalezsosa@suse.com
+
+- Packages module is able to perform a package selection proposal
+  during system upgrade (bsc#1009834)
+- 3.1.120
```

Comment:
In the past (before CASP), when we had a similar situation with changes going to both branches we used to simply do this.
```
- 3.1.121.1 -> A new change that is in both SP2 and CASP
- 3.1.119.4 -> A change that is both in SP2 and CASP
- 3.1.119.3 -> Change added for CASP
- 3.1.119.2 -> Change added for CASP
- 3.1.119.1 -> Change added for CASP
- 3.1.119 -> Here we are in sync with SP2
```
That's more clear to me.